src/App.tsx

```diff
@@ -28,8 +28,8 @@ const App = () => {
                 <DialogBanner/>
                 <UserInfoContext.Provider value={userInfo}>
                     <div className="app__body">
-                        { dialogListe === undefined? null : <DialogOverview dialogData={dialogListe}/> }
-                        { dialogListe === undefined ? null : <Dialog dialog={dialogListe[1]}/> }
+                        {dialogListe === undefined ? null : <DialogOverview dialogData={dialogListe}/>}
```

Comment:
Jeg fikk tidligere dette: Bedre å sjekke for undefined/null i component som kalles; Dialog gjør det allerede for null. Kanksje den bør også gjøre dert for undefined? Begge kan flyttes inn i DialogOverview.

Reply:
Dette har jeg ikke forholdt meg til i denne pull requesten. Disse endringene er kun cmd+alt+L